src/test/resources/SmokeTests.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="SmokeTests">
+
+    <test name="Smoke Test" preserve-order="false">
+        <classes>
+            <class name="homework18"/>
+        </classes>
```

Comment:
In the homework at step,
7. Specify the @Test method to be ran in the new XML file.
Hint: Use the <methods> & <include> tags.
-> You need use <methods> & <include> tags 
//HINT:
```
<classes>
            <class name="homework18">
                <methods>
                    <include name="playSong"/>
                </methods>
            </class>
</classes>
```